README.md

```diff
@@ -20,7 +20,7 @@ bundle exec jekyll serve --drafts
 <details><summary>Development Environment setup - Mac</summary>
 <p>
 
-Are you me? Have you not properly setup ruby on your old macbook? Some of these steps will relate to that issue, ignore if not applicable.
+Are you me? Have you not properly setup ruby on your old mcBook? Some of these steps will relate to that issue, ignore if not applicable.
```

Comment:
🍔

Reply:
haha, intentional, you think it's too much?